components/21-atoms/table/table-mobile.functions.js

```diff
@@ -152,8 +153,8 @@
       addTableList();
     };
 
-    init();
-
-    return {init};
+    if (options.init !== false) {
+      init();
+    }
```

Comment:
So......

* Either you add the option to skip init through the options and **expose** the init function (so it can be called externally)
* OR you call init upon object creation and **don't expose** the init function.

Reply:
You're right :)